src/cluster/__tests__/connectionBuilder.spec.js

```diff
@@ -42,18 +42,18 @@ describe('Cluster > ConnectionBuilder', () => {
     expect(connection.socketFactory).toBe(socketFactory)
   })
 
-  test('when called without host and port iterates throught the seed brokers', () => {
-    const connections = Array(brokers.length)
-      .fill()
-      .map(() => {
-        const { host, port } = builder.build()
-        return `${host}:${port}`
-      })
+  test('when called without host and port picks random seed broker', async () => {
```

Comment:
Isn't this exactly the same as before, except with a for-loop instead of a map?

Reply:
Yes it is except for the loop can handle async/await